web/src/views/sleep_analysis_results/sleep_mechanisms.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { CardDeck, Col, Container } from 'reactstrap';
+
+import FloatingCard from 'components/floating_card';
+
+import './style.css';
+
+const SleepMechanisms = () => (
+  <Container className="text-justify mt-6 mb--300">
+    <CardDeck>
+      <Col lg="6" className="mb-6">
+        <FloatingCard
+          cardClassName="bg-warning"
+          headerText="Circadian Rhythms"
+          bodyText={
+            <span>
+              This is your biological clock. It is a self-sustained internal proccess responsable for regulating your
+              sleep and wake cycles every 24 hours as well as other biological process such as blood pressure, body
+              temperature and heart activity. This is why you feel sleepy at night and awake during the day. Some habits
+              or events may have an influence on these rhythms. Sleep cycles then take place in these broader processes.
+            </span>
+          }
+        />
+      </Col>
+      <Col lg="6" className="mb-6">
+        <FloatingCard
+          cardClassName="bg-primary"
+          headerText="Sleep Homeothesis"
```

Comment:
```suggestion
          headerText="Sleep Homeostasis"
```
Homeostasis ou Homeorhesis? J'imagine que c'est Homeostasis vu le texte en bas.